webbpsf/optics.py

```diff
@@ -1243,37 +1283,15 @@ def __init__(self, instrument, **kwargs):
             instrument,
             **kwargs)
 
-        # TODO load here the wavelength dependence info.
-        self.focusmodel_file = os.path.join(
-            utils.get_webbpsf_data_path(),
-            'NIRCam',
-            'optics',
-            'nircam_defocus_vs_wavelength.fits')
-        model_hdul = fits.open(self.focusmodel_file)
-        assert model_hdul[1].header['XTENSION'] == 'BINTABLE'
-        self.focus_model_data = model_hdul[1].data
-        model_wavelengths, model_defocus = (
-            self.focus_model_data['wavelength'].astype('=f8'),
-            self.focus_model_data['defocus_in_rms_wfe'].astype('=f8')
-        )
-
-        # Read in model data and set up interpolators.
-
-        short_wavelengths_mask = model_wavelengths < 2.45
-        self.fm_short = scipy.interpolate.interp1d(
-            model_wavelengths[short_wavelengths_mask],
-            model_defocus[short_wavelengths_mask],
-            kind='cubic'
-        )
-
-        long_wavelengths_mask = model_wavelengths > 2.45
-        # (n.b. row where model_wavelengths == 2.45 is nan)
-        self.fm_long = scipy.interpolate.interp1d(
-            model_wavelengths[long_wavelengths_mask],
-            model_defocus[long_wavelengths_mask],
-            kind='cubic'
-        )
-
+        # Polynomial equations fit to defocus model (RMS WFE).
```

Comment:
Can you add here please some explanatory comments about where these polynomial coefficient values come from?